--- a/Marlin/Marlin/src/HAL/STM32F1/build_flags.py
+++ b/Marlin/Marlin/src/HAL/STM32F1/build_flags.py
@@ -11,6 +11,7 @@ if __name__ == "__main__":
                     "-fsigned-char",
                     "-fno-move-loop-invariants",
                     "-fno-strict-aliasing",
+                    "-fsingle-precision-constant",

                     "--specs=nano.specs",
                     "--specs=nosys.specs",
